Library/Formula/lysp.rb

```diff
@@ -47,8 +47,8 @@ def install
 +++ b/Makefile
 @@ -1,6 +1,3 @@
 -CFLAGS  = -O  -g -Wall
--CFLAGSO = -O3 -g -Wall -DNDEBUG
--CFLAGSs = -Os -g -Wall -DNDEBUG
+-CFLAGSO = -O3 -g -Wall
```

Comment:
Likewise, here the patch is looking for these lines to remove, so changing them will probably break the patch.